README.md

```diff
@@ -80,70 +104,78 @@ If depositing underlying `asset`:
 
 - The `asset` token will be transferred from the user account to the L1 bridge.
 - The bridge will then deposit the `asset` token in the aToken.
-- A message will be sent to the  L2 bridge with the amount of `staticAToken` to be transferred, the L1 token address and the recipient address as parameters.
-- The token bridge on L2 will then be minting the corresponding staticAToken of the L1 token to the user.
+- A message will be sent to the  L2 bridge with the amount of `static_a_token`
+  to be transferred, the L1 token address and the recipient address as
```

Comment:
`the L1 token address, the recipient address, the block number and the rewards index.`